src/main/java/nl/tudelft/pixelperfect/Game.java

```diff
@@ -1,5 +1,7 @@
 package nl.tudelft.pixelperfect;
 
+import java.io.IOException;
+
```

Comment:
This import is giving me a checkstyle error, it needs to be at the bottom of the input list.